packages/run-protocol/src/psm/psm.js

```diff
@@ -163,7 +164,7 @@ export const start = async (zcf, privateArgs) => {
     }
     seat.exit();
   };
-  const makeSwapInvitation = () => zcf.makeInvitation(swapHook, 'swap');
+  const makeSwapInvitation = () => zcf.makeInvitation(handleSwapOffer, 'swap');
```

Comment:
What's your suggested casing rule? I would have expected either
   * `'CollectFees'` and `'Swap`', or
   * `'collectFees'` and `'swap'`

Reply:
I just neglected to update the string. I _think_ we're better off making string be exactly the chars in `make` … `Invitation` but I don't feel as strongly about it. I could see people wanting to put more descriptive names there and such uniqueness could actually help in a string search of the repo.